### PR TITLE
feat(agoric): implement `agoric run <script> [-- script-args...]`

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

Supply an `args` property to a "deploy" script.  We preserve compatibility by introducing `agoric run` to do this, as an adjunct to `agoric deploy`.

### CODE REVIEW

File: packages/agoric-cli/src/main.js [hunk 1]
```diff
@@ -160,39 +160,63 @@ const main = async (progname, rawArgs, powers) => {
...
     .command('start [profile] [args...]')
-    .description('run an Agoric VM')
+    .description(
+      `\
+start an Agoric VM
+
+agoric start - runs the default profile (dev)
+agoric start dev -- [initArgs] - simulated chain and solo VM
+agoric start local-chain [portNum] -- [initArgs] - local chain
+agoric start local-solo [portNum] [provisionPowers] - local solo VM
```
Comment (dckc) [inline packages/agoric-cli/src/main.js:217]: documenting the start profiles is a bit of a drive-by, but a welcome one...

**Review by dckc — APPROVED** [review]
yay! I have been looking forward to this.

hmm... no corresponding issue. oh well; those don't go in the git history anyway

ideally we'd have more cli tests... perhaps in the documentation repo.
But we didn't have them before this PR, so... onward.

File: packages/agoric-cli/src/main.js (hunk 1 above)
Comment (dckc) [reply packages/agoric-cli/src/main.js:217]: have you looked at docopt, btw?

Comment (michaelfig) [reply packages/agoric-cli/src/main.js:217]: > have you looked at docopt, btw?

Took a look now, but it'll take work to integrate.  Maybe later.
